app/src/main/java/org/sopt/sample/login/LoginRepository.kt

```diff
@@ -0,0 +1,18 @@
+package org.sopt.sample.login
+
+import org.sopt.sample.remote.RequestLoginDTO
+import org.sopt.sample.remote.ServicePool
+import timber.log.Timber
+
+class LoginRepository {
+    suspend fun login(email: String, password: String) : Boolean=
```

Comment:
```suggestion
    suspend fun login(email: String, password: String) : Boolean =
```
Goooood! 코드 정렬만 시켜주세용!